components/custom/Badge/Badge.svelte

```diff
@@ -1,7 +1,11 @@
 <script>
+/** @type {string} the background color of the badge. Defaults to "gray" */
 export let color = 'gray'
+/** @type {string} the border radius of the badge. Defaults to "50%" */
 export let borderRadius = '50%'
+/** @type {string} the padding of the badge. Defaults to ".2em" */
 export let padding = '.2em'
+/** @type {boolean} if the badge is bordered or not */
 export let bordered = false
```

Comment:
It seems like these JSDoc comments are just duplicating the existing code. What is it we gain by adding these?

Reply:
Since they're already there for many of the other files, I won't stand in the way of merging this. I think it would be better to avoid repeating the default value in the comment, though, so that if/when we change any of those later it avoids the risk of contradictory information about the default.